src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/IWindow.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    /// <summary>
+    /// Interface for window
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public interface IWindow
+    {
+        /// <summary>
+        /// Gets window resource ID
+        /// </summary>
+        /// <returns></returns>
+        int GetResId();
+
+        /// <summary>
+        /// Gets window raw pointer
+        /// </summary>
+        /// <returns></returns>
+        IntPtr GetRaw();
```

Comment:
`Handle` is used to represent a native handle
and I recomment to use Property
```suggestion
        IntPtr Handle {get;}
```

Reply:
Removed